blacklisted-modules.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  mainnet: [
+    "0xd6F2E30B99214B6eaffBE2B270d2e42c762a929B", // Disputes v1.0
+    "0x8bEa04A94aEc27CE8C2409b0e0F28A02996d264d", // Registry v1.0
+    "0xF9dDa954adf5E54b89f988C1560553A0a387CCE1", // Registry v1.1
+    "0xA612843B7274384c6F3E78fC78f1EBa22Db26e11", // Voting v1.0
+    "0x3b4c91Cfc006f48344eAFCe8783E46b690d5068F", // Subscriptions v1.0
+  ],
+  rinkeby: [
+    "0x9Ea22B1D388873E230c45C9f140464Dcc48f0efb", // Subscriptions v1.0
```

Comment:
To double check, we don't need to blacklist the other modules in rinkeby and staging because we redeployed them after Feb?

Reply:
Exactly :)